src/AltinnCore/Designer/Controllers/ApplicationMetadataController.cs

```diff
@@ -11,7 +11,7 @@ namespace Designer.Controllers
     /// </summary>
     [ApiController]
     [Authorize]
-    [Route("/designer/api/v1/{org}/{app}")]
+    [Route("/designer/api/v1/{org}/{app}/[controller]")]
```

Comment:
Is the app frontend updated to use this new route? Do not see that change in this pr?

Reply:
The route remains the same since I deleted the route attribute per action. They all had the same attribute with the same value.

Reply:
[ActionName("ApplicationMetadata")]